badge_server/main.py

```diff
@@ -53,7 +53,7 @@
 
 
 @enum.unique
-class BadgeStatus(enum.Enum):
+class BadgeStatus(str, enum.Enum):
```

Comment:
Why inherit from `str` in here?

Reply:
To make `BadgeStatus` serializable when calling `jsonify()` on a dict containing that object:
https://github.com/GoogleCloudPlatform/cloud-opensource-python/blob/bc21101769dd45744c88c04cb82124d875c036d3/badge_server/main.py#L548

I'm not sure if this is the best way to do this, but it was a popular suggestion when I was looking for ways to do this.

Reply:
This LGTM!